packages/design-system/src/styles/components/_Badge.scss

```diff
@@ -2,6 +2,26 @@
 
 $badge-font-size-big: $base-font-size;
 
+@mixin badge-high-contrast-mode-remove-color() {
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    background-color: windowText;
+    color: window;
+    -ms-high-contrast-adjust: none;
+  }
+  /* stylelint-enable scss/media-feature-value-dollar-variable */
+}
+
+@mixin badge-high-contrast-mode-label($label) {
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    &::before {
+      content: $label + ': ';
```

Comment:
Very nice solution!

Reply:
It's a good solution. Do we have an idea for how this could support Spanish, though? Maybe with a [language selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:lang)?

Reply:
@pwolfert Oh, nice catch! I just pushed up an update that can utilize a Spanish language selector.

Reply:
Nice, thank you! I added a comment below. Not sure if the difference between the two selectors was intentional. I could see our users using either `es-MX` or `es-US`. Like `es-US` would probably be more correct for our use cases, but I know of teams who are using `es-MX`.

Reply:
Ah, I got my wires crossed there. I've updated the Spanish language selector to use `^=`, thank you!

Reply:
Cool, LGTM 👍